precompile/assetsbridge/bridge_out.go

```diff
@@ -281,6 +285,32 @@ func (m *BridgeOutMethod) validateToken(
 	}
 }
 
+func (m *BridgeOutMethod) validateAmount(
+	sdkCtx sdk.Context,
+	amount *big.Int,
+	token common.Address,
+) error {
+	sdkAmount, err := precompile.TypesConverter.BigInt.ToSDK(amount)
+	if err != nil {
+		return fmt.Errorf("failed to convert amount: [%w]", err)
+	}
+
+	minAmount := m.bridgeKeeper.GetMinBridgeOutAmount(
+		sdkCtx,
+		token.Bytes(),
+	)
+
+	if sdkAmount.LT(minAmount) {
```

Comment:
There's no point of checking it, but just noting that this only check for < minAmount, but not > to 0, but anyway seeing that the solidity entrypoint is in uint 256, I dont expect any isue here

Reply:
I don't think it's a problem either.
First because `amount` is passed in the `bridgeOut` method as `uint256` and also we have a check
```	
	if inputs.Amount.Sign() <= 0 {
		return errors.New("amount must be positive")
	}
```
ensuring `amount` is positive.